replica/replica.go

```diff
@@ -59,6 +59,9 @@ type Replica struct {
 	peerCache types.PeerDetails
 	peerFile  *sparse.DirectFileIoProcessor
 
+	uuidLock sync.Mutex
```

Comment:
Is it being accessed concurrently ? If yes, can you mention about the same in the comment. I am guessing it's single threaded.

Reply:
No, I saw a lock in revision_counter and thought it's required here also. I will remove it